File: 백준/Bronze/2869. 달팽이는 올라가고 싶다/달팽이는 올라가고 싶다.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner kb = new Scanner(System.in);
        int A = kb.nextInt();
        int B = kb.nextInt();
        int V = kb.nextInt();

        int heightOfDay=A-B;
        int day =0;
        if(A>V){
            day=1;
        }else{
            if((V-A)%heightOfDay==0){
                day = (V-A)/heightOfDay+1;
            }else{
                day=(V-A)/heightOfDay+2;
            }
        }
        System.out.println(day);
        kb.close();
    }
}

File: 백준/Bronze/2869. 달팽이는 올라가고 싶다/달팽이는 올라가고 싶다.cc
#include <stdio.h>
int main(void){
    int A,B,V;
    scanf("%d",&A);
    scanf("%d",&B);
    scanf("%d",&V);
    int height_of_day= A-B;
    int day=0;
    if(A>=V)
        day=1;
    else{
        if((V-A)%height_of_day==0)
            day=(V-A)/height_of_day+1;
        else
            day=(V-A)/height_of_day+2;
    }
    printf("%d",day);
}